salt/metalk8s/macro.sls

```diff
@@ -2,10 +2,10 @@
 
 {%- macro pkg_installed(name='') -%}
   {%- set package = repo.packages[name] | default({}) %}
-  pkg.installed:
+  metalk8s_package_manager.installed:
     - name: {{ name }}
     - fromrepo: {{ repo.repositories.keys() | join(',') }}
-    {%- if package.version | default(None) %}
+    {% if package.version | default(None) %}
```

Comment:
Do not remove the `-` :)

Reply:
FYI: http://jinja.pocoo.org/docs/2.10/templates/#whitespace-control